src/App.tsx

```diff
@@ -15,37 +17,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SetByReverse = true;
```

Comment:
The variable name 'SetByReverse' does not follow the convention for naming constants in JavaScript. Constant names should be in uppercase with underscores separating words. Additionally, the name should be descriptive of its purpose. Consider renaming it to better reflect its role in the application.